SU-SWS/decanter
# ADAPT-000: Move things from TW Utilities to Components

# READY FOR REVIEW

# Summary
- Move things that doesn't need variants from Utilities to Components, so they can be overridden by utilities which are compiled further down the CSS files, e.g., moved `.su-centered-container` to utilities, so I can change its margin using `.su-ml-0` (which is a utility).

# Needed By (Date)
- When does this need to be merged by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Look at preview build, inspect a `div` with the `.su-centered-container` class, e.g. the one with the purple gradient at the top.
1. In the browser inspector, add the class `su-m-0` to this same div
1. It should apply a `margin:0` to the div that isn't being overridden.
1. Look at code.
2. Look at preview page - it should look the same as before.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
